src/cmd_line.rs

```diff
@@ -17,6 +17,8 @@ pub struct CmdLineSettings {
     pub multi_grid: bool,
     pub maximized: bool,
     pub frameless: bool,
+    pub wayland_app_id: std::string::String,
```

Comment:
```suggestion
    pub wayland_app_id: String,
```

You don't need that std::string ..., it's already included in https://doc.rust-lang.org/std/prelude/index.html